cmake/Modules/OpmInit.cmake

```diff
@@ -110,6 +110,10 @@ macro (OpmInitDirVars)
   endif (COMMAND dir_hook)
 endmacro ()
 
+if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "4")
```

Comment:
How about `NOT ("${CMAKE_SIZE_OF_VOID_P}" STREQUAL "8")` instead?

Reply:
Also an option, but does that change justify investing any time?

Reply:
maybe the title/branch/desription was not chosen wisely. I wanted it to fail on 32bit because flow does not work there.

Reply:
i think his point is to check for the condition you actually state you wanted to check, ie, that we are not 64bit, not that we are 32 bit. pretty theoretical, sure..

Reply:
> that we are not 64bit, not that we are 32 bit.

True, that is what I meant.

> pretty theoretical, sure..

Obviously.  I don't expect us to run on 24-bit or 48-bit platforms anytime soon.

On tangential note, however, it is not clear to me why we're using `"${CMAKE_SIZEOF_VOID_P}" STREQUAL` instead of `${CMAKE_SIZEOF_VOID_P} EQUAL` here.  `OpmInit` is supposed to run after `project()` in which case `${CMAKE_SIZEOF_VOID_P}` is known to exist and be a number.

Reply:
Using number comparison now.